library/Fission/Models.hs

```diff
@@ -62,6 +62,7 @@ User
   active        Bool
 
   dataRoot      CID
+  dataRootSize  Natural
```

Comment:
Let’s add a unit!

```haskell
newtype Bytes = Natural
-- And then instances to and from SQL fields. There are examples elsewhere in the code.
```

There’s a nontrivial chance that this already exists in a package. I’ll look it up!

Reply:
Ah, right the fancy version (optional) is something like this:

```haskell
data Bytes  -- No need for a `=` because we only need the type
data KiloBytes
data Meters
data Grams

newtype Quantity unit = Quantity Natural
  deriving newtype (Show, Eq, Integral, AndEvenMoreStuff)

4 :: Quantity Bytes
56 :: Quantity Meters
```